src/__tests__/__snapshots__/node.test.ts.snap

```diff
@@ -319,7 +316,6 @@ Object {
       "40": "40px",
       "5": "5px",
       "8": "8px",
-      "floatingLabelInputFocused": "12px",
```

Comment:
plus one. these don't make much sense as "named" ones. we started out naming magic numbers. by now a lot of them are just regular values (we have way too many of them anyway). plus we have css modules for true one-offs. good call!